test/recipes/25-test_req.t

```diff
@@ -389,6 +389,18 @@ subtest "generating certificate requests with ML-DSA" => sub {
                     "-subj", "/CN=test-self-signed",
                     "-addext","keyUsage=digitalSignature"])),
                     "Generating self signed ML-DSA-87 cert and private key");
+        ok(run(app(["openssl", "req",
+                    "-config", srctop_file("test", "test.cnf"),
+                    "-new",
+                    "-sigopt","hextest-entropy:000102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f",
+                    "-out", "csr_ml_dsa_87.pem",
+                    "-newkey", "ML-DSA-87",
```

Comment:
Why only ML-DSA-87?

Reply:
The first commit is not part of this PR.. Can add to the other PR if required.